bench/wtperf/wtperf.h

```diff
@@ -192,7 +192,8 @@ struct __wtperf {         /* Per-database structure */
 
     volatile bool idle_cycle_run; /* Signal for idle cycle thread */
 
-    volatile uint32_t totalsec; /* total seconds running */
+    volatile uint32_t totalsec; /* total seconds running - including populate */
+    uint32_t testsec;           /* total seconds executing workload */
```

Comment:
Two minor nits: First alphabetize `testsec` before `totalsec`. Second, should it also be volatile like the others around it given its similar usage?

Reply:
I fixed the alphabetization. 

The other fields are volatile because they are read by the monitor thread. The new field is only set at the end of the test. So it's not of interest to the monitor and doesn't need to be volatile.